test_ros2cli/test/config_ros2launch_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
I guess this file was mistakenly added? (And `config_ros2service_test.py` too).

Reply:
Yes, that is a half baked work that I pushed by error. Removed them in https://github.com/ros2/ros2cli/pull/287/commits/6b684d2fcd29adce3e093ace7eb3d72a11347fe0.
They will be part of a follow-up PR.